src/gui/optionsdlg.ui

```diff
@@ -213,6 +213,16 @@
                 </widget>
                </item>
                <item>
+                <widget class="QCheckBox" name="checkLargeStatusIcons">
```

Comment:
I thought about that, but the transfer list UI options seemed like an obvious place for it. It's about as significant as hiding zero and infinity values IMO.